components/resources/wallet_strings.grdp

```diff
@@ -291,6 +292,10 @@
     <message name="IDS_BRAVE_WALLET_EDIT_GAS_GWEI" desc="Edit gas panel gwei label">Gwei</message>
     <message name="IDS_BRAVE_WALLET_EDIT_GAS_SET_CUSTOM" desc="Edit gas panel set custom button">Set custom</message>
     <message name="IDS_BRAVE_WALLET_EDIT_GAS_SET_SUGGESTED" desc="Edit gas panel set suggested button">Set suggested</message>
+    <message name="IDS_BRAVE_WALLET_NOT_VALID_ETH_ADDRESS" desc="Send input not valid ETH address error">Not a valid ETH address</message>
+    <message name="IDS_BRAVE_WALLET_NOT_VALID_ADDRESS" desc="Send input not valid address error">Not a valid address</message>
+    <message name="IDS_BRAVE_WALLET_NOT_DOMAIN_FIRST" desc="Send input not registered domain first error">Domain</message>
+    <message name="IDS_BRAVE_WALLET_NOT_DOMAIN_SECOND" desc="Send input not registered domain second error">is not registered</message>
```

Comment:
You can do the following:

```
Domain <ph name="VALUE">$1</ph> is not registered
```

and in the React components, do `getLocale('braveWalletNotDomain').replace('$1', url)`.

Reply:
Changed! Works nice!